frontend/public/locales/en/translation.json

```diff
@@ -2090,7 +2090,8 @@
     "submariner.install.form.gateways.labelHelp": "The number of worker nodes  that will be used to deploy the Submariner gateway component on the managed cluster. If the value is greater than 1, the Submariner gateway HA will be enabled automatically. (default 1)",
     "submariner.install.form.gateways.placeholder": "Enter gateway count",
     "submariner.install.form.instancetype": "Instance type",
-    "submariner.install.form.instancetype.labelHelp": "The Amazon Web Services EC2 instance type of the gateway node that will be created on the managed cluster. (default c5d.large)",
+    "submariner.install.form.instancetype.labelHelp.aws": "The Amazon Web Services EC2 instance type of the gateway node that will be created on the managed cluster. (default c5d.large)",
+    "submariner.install.form.instancetype.labelHelp.az": "The Azure instance type of the gateway node that will be created on the managed cluster. (default Standard_F8s_v2)",
```

Comment:
Should this be "Standard_F4s_v2"?

Reply:
ya that is right, this value was something I used for testing , Changed to  Standard_F4s_v2